inference_perf/client/filestorage/gcs.py

```diff
@@ -12,18 +12,21 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 import json
+import logging
 from typing import List
 from google.cloud import storage
 from google.cloud.exceptions import GoogleCloudError
 from inference_perf.client.filestorage import StorageClient
 from inference_perf.config import GoogleCloudStorageConfig
 from inference_perf.utils import ReportFile
 
+logger = logging.getLogger(__name__)
+
 
 class GoogleCloudStorageClient(StorageClient):
     def __init__(self, config: GoogleCloudStorageConfig) -> None:
         super().__init__(config=config)
-        print("Created new GCS client")
+        logger.info("Created new GCS client")
```

Comment:
debug level?

Reply:
Thanks for your review! I updated the code in [76c1b70](https://github.com/kubernetes-sigs/inference-perf/pull/110/commits/76c1b703f85d3608c617290dea2933b16c2bcd90).